### DISCUSSION TITLE
Can I import the browser bookmark into this?

### DISCUSSION THREAD

**Discussion by acgxv** [zombieFox/nightTab#380] [Q&A]
Can I import the browser bookmark into this?

If I have to add the bookmarks one by one, that's painful.

Or can it read the browser bookmark automatically?

**Comment by medusahra** [comment]
> Can I import the browser bookmark into this?
> 
> If I have to add the bookmarks one by one, that's painful.
> 
> Or can it read the browser bookmark automatically?

apparently it doesn't let you import anything, you have to put it one by one which is ridiculous, but that's how it works for now.
